frontend/src/components/dashboard/EditProfile.tsx

```
@@ -1,22 +1,301 @@
+              <h3 className="text-base font-medium mb-2">Education</h3>
+
+              {educations.map((education) => (
+                <div key={education.id} className="mb-6">
+                  <div className="grid grid-cols-1 md:grid-cols-[1fr,auto,1fr,auto,1fr] gap-2 items-center mb-2">
+                    <Input placeholder="Degree" className="h-12 rounded-md" />
+                    <span className="hidden md:inline text-center">at</span>
+                    <Input
+                      placeholder="School/University"
+                      className="h-12 rounded-md"
+                    />
+                    <span className="hidden md:inline"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)